website_multi_theme/README.rst

```diff
@@ -117,7 +117,7 @@ Known issues / Roadmap
 * Theme picker should include some kind of thumbnail if possible.
 * If you install any of the supported themes after installing this addon, you
   will have to press *Reload* in the website config wizard to make it notice
-  the change.
+  the change. The same is true when you install any module with ``customize_show="True"``
```

Comment:
Tests are not passed in travis due to this new requirement: *Reload* must be done after installing  website_blog_category https://github.com/OCA/website/blob/10.0/website_blog_category/templates/website_blog_template.xml#L25

What can we do here? @Yajo @pedrobaeza

Reply:
Do you mean demo file of **website_blog_category**?

Reply:
I'd prefer to Reload after creating new ``ir.ui.view`` record with ``customize_show`` (via extending ``create`` method). What do you think, @pedrobaeza ?

Reply:
That would make sense, yes.

I was talking about demo data file included in this addon. Or even maybe a normal data file.